docs/contributing/documentation/admins.md

```diff
@@ -84,49 +84,42 @@ To make it easier for other contributors to work with your project, update the f
-1.  Under {guilabel}`Build & deploy > Continuous deployment`, scroll to {guilabel}`Branches and deploy contexts`, and click {guilabel}`Configure`, then enter appropriate values.
+## Add a project to Read the Docs
+
+To add a new site to Read the Docs to preview documentation or storybooks in pull requests, you need to configure your project's repository and import it into Read the Docs.
+You also need an account on Read the Docs and have write access to the repository.
+
+
+### Configuration files
+
+The following are example files that you can use to configure your project for pull request previews on Read the Docs.
+
+-   [Plone Sphinx Theme `Makefile`](https://github.com/plone/plone-sphinx-theme/blob/main/Makefile), specifically the `rtd-pr-preview` section.
+    This is the command to use to build documentation previews on Read the Docs.
+-   [Plone Sphinx Theme `requirements-initial.txt`](https://github.com/plone/plone-sphinx-theme/blob/main/requirements-initial.txt) specifies the initial Python packaging tool requirements to set up a virtual environment.
+-   [Plone Sphinx Theme `requirements-docs.txt`](https://github.com/plone/plone-sphinx-theme/blob/main/requirements-docs.txt) specifies the requirements to use Plone Sphinx Theme and build the docs.
+-   [Plone Sphinx Theme `conf.py`](https://github.com/plone/plone-sphinx-theme/blob/main/docs/conf.py) the Sphinx configuration file to build the docs.
+-   [Plone Sphinx Theme `.readthedocs.yaml`](https://github.com/plone/plone-sphinx-theme/blob/main/.readthedocs.yaml) specifies the configuration and command to build the docs.
+-   [Plone Sphinx Theme `.github/workflows/rtd-pr-preview.yml`](https://github.com/plone/plone-sphinx-theme/blob/main/.github/workflows/rtd-pr-preview.yml) specifies when to build the docs, specifically only when a pull request is opened against the `main` branch and there are changes to documentation files.
+    You might need to adjust the branch name, paths, and files to check for changes.
+
+
+### Read the Docs administration
+
+After logging in to your Read the Docs account, you can import your project.
+
+1.  Click {guilabel}`Add project`.
+1.  For {guilabel}`Repository name`, enter the GitHub organization, a forward slash, and the repository to import, for example, `plone/volto`.
+1.  Click {guilabel}`Continue`.
+1.  In the {guilabel}`Add project` screen, you can configure basic project settings, including its {guilabel}`Name`, {guilabel}`Repository URL`, {guilabel}`Default branch`, and {guilabel}`Language`.
+    The defaults are usually accurate.
+1.  Click {guilabel}`Next`.
+1.  A sample `.readthedocs.yaml` file is suggested, if you have not already added one.
+1.  Click {guilabel}`Finish`.
+    Read the Docs will redirect you to the project details, and start building the docs.
+
+For most Plone projects, you will not want to Read the Docs to publish the `latest` or other specific versions.
+Plone projects currently self-host their official documentation.
```

Comment:
Sorry, I do not get exactly what this two lines are supposed to tell me.

Reply:
We already have a public version that gets indexed by search engines. We don't want that for pull request previews. I also realized this section was kind of incomplete in regards to activating and hiding versions. Here's a revision.

````suggestion
Many Plone projects currently self-host their official documentation at {doc}`/index`.
These projects get indexed by search engines.

For pull request previews, unsupported branches or versions, or other situations, you most likely do not want search engines to index your documentation.
In this case, you can create a custom {file}`robots.txt` file to discourage, but not absolutely prevent, search engine indexing.
You can configure Sphinx to copy the {file}`robots.txt` file.
However, if you want to have two versions of a {file}`robots.txt` file—say one that allows indexing of your official documentation and another that discourages indexing—you can configuration your automation to copy it into place.

```shell
cp some-path/block-robots.txt docs-root-path/robots.txt
```

You can also configure which branches to activate or hide through Read the Docs.

```{seealso}
-   [`robots.txt` support](https://docs.readthedocs.io/en/stable/reference/robots.html)
-   [Automation rules](https://docs.readthedocs.io/en/stable/automation-rules.html)
-   [Versions](https://docs.readthedocs.io/en/stable/versions.html)
-   [Managing versions automatically](https://docs.readthedocs.io/en/stable/guides/automation-rules.html)
```
````

Reply:
@jensens I revised this section and improved the flow. Please let me know.